src/commands/database-rules-release.ts

```diff
@@ -0,0 +1,23 @@
+import { Command } from "../command";
+import * as logger from "../logger";
+import * as requireInstance from "../requireInstance";
+import { requirePermissions } from "../requirePermissions";
+import * as metadata from "../database/metadata";
+
+export default new Command("database:rules:release <ruleset_id>")
+  .description("mark a staged ruleset as the stable ruleset")
+  .option(
+    "--instance <instance>",
+    "use the database <instance>.firebaseio.com (if omitted, uses default database instance)"
+  )
+  .before(requirePermissions, ["firebasedatabase.instances.get"])
```

Comment:
Does this really only need the `instances.get` permission?

Reply:
changing this to `firebasedatabase.instances.update`, good call